examples/browser-cors/index.js

```diff
@@ -1,14 +0,0 @@
-
-/**
- * Module dependencies.
- */
-
-var path = require('path');
-var express = require('express');
-
-express()
-.use(express.static(path.resolve(__dirname, '..', '..', 'dist')))
-.use(express.static(__dirname))
-.listen(3000);
-
-console.log('wpcom app started on port 3000');
```

Comment:
This was from a commit by @retrofox https://github.com/Automattic/wpcom.js/commit/accc3d3bc856e77bafbbc7280f56c4f06269ccda